.github/ISSUE_TEMPLATE/public-websites-intake.md

```diff
@@ -10,57 +10,64 @@ assignees: karak888
 # WIP
 
 ### Instructions
-- All requests for work needing to be fulfilled by a Public Websites resource must submit this intake request form. The team's capacity and the type of request will determine which sprint the work falls into, so it is highly recommended that requests are submitted as soon as you're aware you will need help. Different types of requests may require more time in advance.
+- All teams requesting support or work that needs to be fulfilled by a Public Website resource must submit this intake request form. The team's capacity and the type of request will determine which sprint the work falls into.
+
+__When to submit request a ticket__
+Make your request as soon as you're aware you will need public website participation. Different types of requests may require more time in advance. For apps tool content support, try to engage a content person for your project kick-off.
 
 ### Description
 *Detailed description of the request including background information and reason for the request*
 
-### Type of request
-*Select one*
-- [ ] Tool / Product 
-
-      1. Is this related to a VA.gov tool? 
-      2. If yes provide brief description of tool 
-      3. Is this an iteration to an existing tool?
-      4. Is there already a particular content person assigned for the tool, if so please provide name.
-      5. Provide any links to existing wireframes, prototype 
-      6. Provide any links to product briefs or outlines
-      7. Is there an ETA for launch?
-- [ ] Shared content resource 
-
-      1. Describe reason / need for content resource 
-      2. How long will you require resource?
+ 1. Is the request related to a VA.gov tool? If yes provide brief description of the tool or funcitionality: 
+ 3. Is this an iteration to an existing tool?  [Not sure why this detail is needed - ??? let's verify w/FE and Content if this detail is used.] 
+ 5. Provide a link to product brief or outline: 
+ 6. Provide a link to wireframes, prototype: 
+ 7. Provide a link to any user research synthesis: 
+
+
+### Desired/target implementation date
+Is there an ETA for a particular launch or completion date when request needs to be completed (for instance, for a user research session or eBenefits migration)?
+
+
+### Type of support request
```

Comment:
Consider moving this right under the types of requests Pubic Websites support so it's immediately clear what folks are asking the team.